go-controller/pkg/util/ovs.go

```diff
@@ -766,3 +766,14 @@ func DetectSCTPSupport() (bool, error) {
 	}
 	return false, nil
 }
+
+// Finds any OVN Load Balancer based on external ID and value
+func FindOVNLoadBalancer(externalID, externalValue string) (string, string, error) {
+	out, stderr, err := RunOVNNbctl("--data=bare",
+		"--no-heading", "--columns=_uuid", "find", "load_balancer",
+		"external_ids:"+externalID+"="+externalValue)
+	if err != nil {
+		return "", stderr, err
+	}
+	return out, "", nil
+}
```

Comment:
we have a package for OVN loadbalancers operations, better move this there https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/ovn/loadbalancer/loadbalancer.go#L101

Reply:
ahh, ic, you added some of the functions ther, any special reason to have this here?

Reply:
Honestly I don't remember, but I think it was because the function is used by both the loadbalancer and gateway packages and I was worried about a current/future cyclical dependency